cli/packages/prisma-client-lib/src/codegen/generators/go-client.ts

```diff
@@ -267,7 +267,37 @@ export class GoGenerator extends Generator {
 
                     var v Aggregate
                     _, err := ret.Exec(ctx, &v)
-                    return v, err
+                    return &v, err
+                  }`
+                )
+              }
+              if (
+                type.name.endsWith('Connection') &&
+                field.name === 'edges'
+              ) {
+                // edges is a special case where a field `node` has nested fields
+                const objectName = type.name.replace('Connection', '')
+                return (
+                  sTyp +
+                  `
+                  func (instance *${goCase(type.name)}Exec) ${goCase(
+                    field.name,
+                  )}() *${objectName}EdgeExecArray {
+                    edges := instance.exec.Client.GetMany(
+                      instance.exec,
+                      nil,
+                      [3]string{"${objectName}WhereInput", "${objectName}OrderByInput", "${objectName}Edge"},
+                      "edges",
+                      []string{"cursor"})
+
+                    nodes := edges.Client.GetMany(
+                      edges,
+                      nil,
+                      [3]string{"", "", "${objectName}"},
+                      "node",
+                      []string{"id", "createdAt", "updatedAt", "name", "desc"})
```

Comment:
I am not sure why name and desc are required table columns?

Reply:
This was by accident, it's fixed in #4730

Reply:
Ah, thank you @steebchen for the explanation.